packages/webdriverio/src/commands/browser/saveScreenshot.js

```diff
@@ -32,7 +32,7 @@ export default async function saveScreenshot (filepath) {
         throw new Error('saveScreenshot expects a filepath of type string and ".png" file ending')
     }
 
-    const absoluteFilepath = filepath.startsWith('/')
+    const absoluteFilepath = filepath.startsWith('/') | filepath.startsWith('\\')
```

Comment:
It should be a proper _OR_
```js
const absoluteFilepath = filepath.startsWith('/') || filepath.startsWith('\\')
```

Reply:
Yeah, somehow I missed that it wasn't, will update